docs/config-system.md

```diff
@@ -6,15 +6,15 @@ configuration system in Clusterio, and unless you're developing
 Clusterio it's probably not going to be very useful for you.</sub>
 
 Clusterio uses a group based configuration system to manage settings for
-the master server, instances, slaves, and the clusterctl utility.
+the master server, instances, slaves, and the clusterioctl utility.
 Plugins get their own config group for the master server and instances
 which they can add their own fields to.  The fields are defined early in
 the startup, and the groups and their fields for disabled plugins will
 still be created.
 
 The built-in config groups and their fields are defined in
-[/lib/config/definitions.js](lib/config/definitions.js) while plugins
-can add their own config groups to the `MasterConfigGroup` and
+[lib/config/definitions.js](/packages/lib/config/definitions.js) while
```

Comment:
Same here. This is developer documentation, I don't think its a given that a new developer would know to look in /packages